sdk/servicebus/service-bus/test/internal/node/packageInfo.spec.ts

```diff
@@ -1,17 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import _default from '@opentelemetry/api';
```

Comment:
What's happening here?

Reply:
I think this also I had added for debug purpose. don't remember

Reply:
Removed it